boot/boot.js

```diff
@@ -498,10 +503,32 @@ $tw.utils.getTypeEncoding = function(ext) {
 	return typeInfo ? typeInfo.encoding : "utf8";
 };
 
+var polyfill =[
+	"// this polyfills the globalThis variable",
+	"// using the this variable on a getter ",
+	"// inserted into the prototype of globalThis",
+	"(function() {",
+	"	if (typeof globalThis === 'object') return;",
+	"	// node.green says this is available since 0.10.48",
+	"	Object.prototype.__defineGetter__('__wow__', function() {",
```

Comment:
We generally use `Object.defineProperty()`; is there a reason for using `__defineGetter__` here?

Reply:
Yes, this part needs to be exactly as written. It's probably needed for backward compatibility with IE or something.